### PR TITLE
Fix #70: Show y values in table below chart

### LINKED ISSUE #70: Display both "c" and "p" on hover on charts
Currently we only show the proportion but seeing the cumulative on hover would more easily allow someone to say "50% of users have <X> or less <metric>".

### PR DESCRIPTION
**Pull request by openjck**

### CODE REVIEW

**Comment by openjck** [comment]
I guess I'm not totally in love with this design. I considered putting everything on one line without labels, but it wouldn't all fit comfortably. Suggestions welcome.

File: distributionviewer/core/static/js/app/components/views/chart.js [hunk 1]
```diff
@@ -49,7 +49,20 @@ export class Chart extends React.Component {
     const chart = (
       <div className={`chart is-fetching chart-${this.props.chartId}`}>
         <Fetching />
-        <p className="chart-rollover-container"><span /><span /></p>
+        <table className="chart-rollover-table">
```
Comment (spasovski) [inline distributionviewer/core/static/js/app/components/views/chart.js:52]: We could've also gone with a `<dl>`. Feel free to ignore this a table is quite fine too.

File: distributionviewer/core/static/js/app/components/views/chart.js [hunk 2]
```diff
@@ -113,8 +126,9 @@ export class Chart extends React.Component {
       yax_count: 5,
       mouseover: data => {
         infoElm.classList.add('show');
-        infoElm.querySelector('span').textContent = refLabels[data.x];
-        infoElm.querySelector('span:last-child').textContent = `${data.p.toFixed(4)}%`;
+        infoElm.querySelector('.value-x').textContent = refLabels[data.x];
```
Comment (spasovski) [inline distributionviewer/core/static/js/app/components/views/chart.js:129]: Thanks! The class selectors make it far more readable.

**Comment by spasovski** [comment]
Looks good but it makes it so we have to leave lots of space. I'd vote for moving the `infoElm` to live above `.chart-menu` and only have 1 but might as well do it in another commit.

**Comment by openjck** [comment]
> Looks good but it makes it so we have to leave lots of space. I'd vote for moving the infoElm to live above .chart-menu and only have 1 but might as well do it in another commit.

Agreed. I think that's a great idea.

**Pull request merged**